as-controller-board-status/functions/output.go

```diff
@@ -246,15 +248,49 @@ func (boardStatus *CheckBoardStatus) processVendingDoorState(lc logger.LoggingCl
 			return fmt.Errorf("failed to submit the controller board's status to the central vending state service: %v", err.Error())
 		}
 
-		// Prepare a message to be sent to the MQTT bus. Depending on the state
-		// of the door, this message may trigger a CV inference
-		err = boardStatus.RESTCommandJSON(boardStatus.Configuration.DoorStatusCommandEndpoint, http.MethodPut, VendingDoorStatus{
```

Comment:
RESTCommandJSON in common.go needs to be removed (no longer used). Since it is the only function in common.go, the file and test file also should be removed.

Reply:
But it is still used - https://github.com/intel-retail/automated-vending/blob/23a2dfdf004f6b778a25450018a4dec5eb9cf73c/as-controller-board-status/functions/output.go#L242

Reply:
Ohh, ok. Used for the AV endpoints.